eval/src/tests/instruction/dense_fast_rename_optimizer/dense_fast_rename_optimizer_test.cpp

```diff
@@ -59,8 +60,12 @@ TEST("require that transposing dense renames are not optimized") {
     TEST_DO(verify_not_optimized("rename(x5y3,(y,x),(a,b))"));
 }
 
-TEST("require that non-dense renames are not optimized") {
-    TEST_DO(verify_not_optimized("rename(x_m,x,y)"));
+TEST("require that non-dense renames may be optimized") {
```

Comment:
should also test with mixed tensor multi-transpose to verify that mapped and indexed dimensions are independent of each other.